packages/wish_deposit/lib/src/wish_deposit.dart

```diff
@@ -0,0 +1,12 @@
+import 'package:deposit/deposit.dart';
+import 'package:wish_deposit/wish_deposit.dart';
+
+/// {@template wish_deposit}
+/// A deposit that talks to the wishes data backend.
+/// {@endtemplate}
+class WishDeposit extends Deposit<WishEntity, int> {
+  /// {@macro wish_deposit}
+  WishDeposit({
+    required DepositAdapter<int> adapter,
```

Comment:
Will this adapter be implemented in a separate pull request?

Reply:
I am planning on using supabase for the whole project, so instead of defining a constructor that can initialise a supabase adapter I thought it might be more better to just pass it to the deposit. That way I can define a single adapter on the app project and pass it along to any deposit that will need it.